index.js

```diff
@@ -4,7 +4,7 @@ const requestBuilder = require('./requestBuilder');
 const Parsers = require('./parsers');
 
 class Darwin {
-  constructor(apiKey, options) {
+  constructor(apiKey, options = {}) {
```

Comment:
Some of the methods were failing unless an empty options object was provided, so for consistency I've added it to all of them